examples/app_multi_node/README.md

```diff
@@ -6,36 +6,46 @@ Lightning supports makes multi-node training simple by providing a simple interf
 
 You can run the multi-node raw PyTorch by running the following commands.
 
+Here is an example where you setup spawn your processes yourself.
+
+```bash
+lightning run app train_pytorch.py
+```
+
+or you can use the built-in component for it.
+
 ```bash
-lightning run app app_torch_work.py
+lightning run app train_pytorch_spawn.py
 ```
 
 ## Multi Node with raw PyTorch + Lite
 
 You can run the multi-node raw PyTorch and Lite by running the following commands.
 
+This removes all the boilerplate around distributed strategy by you remain in control of your loops.
```

Comment:
something wrong with the sentence at the end

Reply:
Updated to: `Using Lite, you retain control over your loops while accessing in a minimal way all Lightning distributed strategies.`